rotkehlchen/api/v1/schemas.py

```diff
@@ -853,6 +855,7 @@ def transform_data(  # pylint: disable=no-self-use
             pnl_csv_have_summary=data['pnl_csv_have_summary'],
             ssf_0graph_multiplier=data['ssf_0graph_multiplier'],
             non_syncing_exchanges=data['non_syncing_exchanges'],
+            pnl_order=PnlOrder.deserialize(data['pnl_order']) if data['pnl_order'] is not None else None,  # noqa: E501
```

Comment:
don't do that here. Just use a `SerializableEnumField` for the attribute.

Reply:
Ok